src/index.html

```diff
@@ -17,6 +21,94 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="header-container">
+        <div class="logo-container">
+          <a href="#">
+            <img
+              src="./images/logo.png"
+              alt="MOYO Logo"
+              class="logo"
+            />
+          </a>
+        </div>
+        <nav class="nav__container">
+          <ul>
+            <li>
+              <a
+                class="nav__link is-active"
+                href="#"
+              >
```

Comment:
The 'href' attributes in these 'a' tags should not have spaces. Anchor links should start with the '#' symbol.